src/domains/ecommerce-common/03_storage.tf

```diff
@@ -115,8 +115,8 @@ resource "azurerm_storage_queue" "transactions_authorization_requested_queue" {
   storage_account_name = module.ecommerce_storage_transient.name
 }
 
-resource "azurerm_storage_queue" "transactions_authorization_requested_retry_queue" {
-  name                 = "${local.project}-transaction-auth-requested-retry-queue"
+resource "azurerm_storage_queue" "transactions_authorization_outcome_waiting_queue" {
```

Comment:
maybe here we have to duplicate queue leaving the old one so that first apply will only add the new queue then proceed with modules deploy and remove the old queue in a separate pr. What do you think?

Reply:
This queue is not referenced in any services deployed in production. We can proceed to replace it directly. Anyway we can evaluate a gradual approach to avoid apply failure due to lock